server/events/vcs/github_client.go

```diff
@@ -273,6 +338,34 @@ func (g *GithubClient) PullIsApproved(repo models.Repo, pull models.PullRequest)
 	return approvalStatus, nil
 }
 
+func (g *GithubClient) DiscardReviews(repo models.Repo, pull models.PullRequest) error {
+	reviewStatus, err := g.getPRReviews(repo, pull)
+	if err != nil {
+		return err
+	}
+
+	var mutation struct {
+		DismissPullRequestReview struct {
+			PullRequestReview struct {
+				ID githubv4.ID
+			}
+		} `graphql:"dismissPullRequestReview(input: $input)"`
+	}
+	for _, review := range reviewStatus.Reviews {
```

Comment:
Is there not a way to dismiss all reviews, without having to retrieve them all, and using a single call?

Reply:
If there is one, I haven't been able to find one.   
There is only a GraphQL verb for dismissal a single review.

Reply:
Ok, understood.

The reason I ask is because we want to limit the VCS API calls as much as possible to avoid getting throttled and having to add additional retry logic.

Please add this your comment as a comment in the code for future readers. Perhaps they will add a way in the future, someone will see the comment in the code, and PR it forward.